llvm/test/Analysis/CtxProfAnalysis/load-unapplicable.ll

```diff
@@ -0,0 +1,60 @@
+; REQUIRES: x86_64-linux
+;
+; RUN: rm -rf %t
+; RUN: split-file %s %t
+; RUN: llvm-ctxprof-util fromJSON --input=%t/profile.json --output=%t/profile.ctxprofdata
+; RUN: opt -passes='require<ctx-prof-analysis>,print<ctx-prof-analysis>' -ctx-profile-printer-level=everything \
+; RUN:   %t/example.ll -S 2>&1 | FileCheck %s
+
+; CHECK: No contextual profile was provided
+;
+; This is the reference profile, laid out in the format the json formatter will
+; output it from opt.
+;--- profile.json
+[
+  {
+    "Counters": [
+      9
+    ],
+    "Guid": 12341
```

Comment:
I take it none of the GUIDs here match the code snippet below?

Reply:
I suggest just adding a comment that the GUIDs are explicitly set to different values than the one in the code so that it doesn't match.

Reply:
Just saw that now. Added in 1d2da210a2f0. Also made the GUID values more obvious.